pipelines/ops/druid/Jenkinsfile.ingestion

```diff
@@ -25,8 +25,22 @@ node("ops-slave") {
                 module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
                 jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
                 currentWs = sh(returnStdout: true, script: 'pwd').trim()
+        // populating the vars for raw-overlord and rollup-overlord cluster respectively.
+                list_vars="$ingestion_task_names";
+                println(list_vars);
+                List<String> list_vars_array = list_vars.split(',');
+                List<String> rollup_vars = [], raw_vars = [];
+                list_vars_array.each {
+                    if(it.startsWith("rollup")) {
+                        rollup_vars.add(it);
+                    }
+                    else
+                       raw_vars.add(it);
+                }
+                rollup_ingestion_task_names = rollup_vars.join(",");
```

Comment:
We dont need additonal variables `rollup_ingestion_task_names` and `raw_ingestion_task_names`, you can remove this and the next line.

Reply:
updated as suggested.